video_emotion/extract_classifier.py

```diff
@@ -1,5 +1,6 @@
 import sys
-
+import os
+sys.path.insert(0, os.path.abspath("../.."))
```

Comment:
Is it on purpose that this line is different from that in "extract_test_data.py"
If so then ignore this